js/nathan.js

```diff
@@ -2,5 +2,22 @@
 // returns the min in the array
 function min(array) {
     console.log("stub for min");
-    return 35;
+
+    //Variable...
+    let x = 0;
+
+    //Pull min number from array...
+
+    for(i=0; i<array.length-1;i++)
+    {
+        if(i==0){
+            x=array[i];
```

Comment:
could you just do this line somewhere else and then not need a nested if statement. it would be simpler.